.github/workflows/build-centraldashboard.yml

```diff
@@ -45,12 +45,7 @@ jobs:
         docker tag ${{ env.REGISTRY_NAME }}.azurecr.io/kubeflow/centraldashboard:${{ github.sha }} \
           ${{ env.REGISTRY_NAME }}.azurecr.io/kubeflow/centraldashboard:${GITHUB_REF#refs/*/}
         docker push ${{ env.REGISTRY_NAME }}.azurecr.io/kubeflow/centraldashboard:${GITHUB_REF#refs/*/}
-        docker system prune -f -a
```

Comment:
Should this be entirely removed, or just moved to the end of file?

Reply:
@wg102 I don't think there is actually any need for pruning here, nor later in the script.